whylog/front/utils.py

```diff
@@ -0,0 +1,19 @@
+LOCALHOST = '127.0.0.1'
+
+
+class FrontInput(object):
+    def __init__(self, offset, line_content, resource_location):
+        self.offset = offset
+        self.line_content = line_content
+        self.resource_location = resource_location
+
+
+class AbstractResourceLocation(object):
+    def __init__(self, path, host):
```

Comment:
some child classes like `LocallyAccessibleLogOpener` don't have an explicitly defined host. Generally resource locations don't need to have a host - they can use service discovery or something like that, to access the log service. I suggest that those two arguments are removed from the interface of `AbstractResourceLocation`.

Also, `AbstractResourceLocation` is not really an abstract class (`@six.add_metaclass(ABCMeta)` is missing)